### ISSUE TITLE
[Bug] Conflict with KalielsTracker

### ISSUE THREAD

**Issue by ZINK-ZINK** [Gethe/Aurora#16]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Client

Retail

### Current Behavior

Just switched from Elvui to Aurora+Grid2+Raven+other addons I've had for ages trying to use less system resources. Loving the new setup a lot more and went from 45fps to 100fps in raids. I can't get this Aurora bug to stop though, after a Bugsack wipe the error pops even on a fresh exit and restart of WoW the moment I load into game. It doesn't seem to be a problem, but errors are never great. Here's the Bugsack:

1x ...\Skin\SharedXML\SharedUIPanelTemplates.lua:386: attempt to index local 'Frame' (a nil value)
[string "@Aurora\Skin\SharedXML\SharedUIPanelTemplates.lua"]:386: in function `NineSlicePanelTemplate'
[string "@Aurora\Skin\SharedXML\SharedUIPanelTemplates.lua"]:495: in function `TooltipBackdropTemplate'
[string "@!KalielsTracker\Libs\MSA-DropDownMenu-1.0\MSA-DropDownMenu-1.0-12.lua"]:1704: in function <...r\Libs\MSA-DropDownMenu-1.0\MSA-DropDownMenu-1.0.lua:1700>
[string "@!KalielsTracker\Libs\MSA-DropDownMenu-1.0\MSA-DropDownMenu-1.0-12.lua"]:1714: in function <...r\Libs\MSA-DropDownMenu-1.0\MSA-DropDownMenu-1.0.lua:1711>

Locals:
Frame = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'Frame' (a nil value)"
Hook = <table> {
 LFDQueueFrameRandomCooldownFrame_Update = <function> defined @Aurora\Skin\FrameXML\LFDFrame.lua:13
 GuildMicroButtonMixin = <table> {
 }
 ChallengesKeystoneFrameMixin_OnKeystoneSlotted = <function> defined @Aurora\SkinBlizzard_ChallengesUI.lua:26
 QuestObjectiveSetupBlockButton_AddRightButton = <function> defined @Aurora\SkinBlizzard_ObjectiveTracker.lua:21
 FCF_SetButtonSide = <function> defined @Aurora\Skin\FrameXML\FloatingChatFrame.lua:40
 QuestFrameProgressItems_Update = <function> defined @Aurora\Skin\FrameXML\QuestFrame.lua:13
 OrderHallMission = <table> {
 }
 HelpPlate_GetButton = <function> defined @Aurora\Skin\FrameXML\TutorialFrame.lua:69
 LFGListApplicationViewer_UpdateRoleIcons = <function> defined @Aurora\Skin\FrameXML\LFGList.lua:91
 ChallengesKeystoneFrameMixin_Reset = <function> defined @Aurora\SkinBlizzard_ChallengesUI.lua:22
 EmbeddedItemTooltip_PrepareForSpell = <function> defined @Aurora\Skin\FrameXML\GameTooltip.lua:23
 NavBar_CheckLength = <function> defined @Aurora\Skin\FrameXML\NavigationBar.lua:17
 BossBanner_ConfigureLootFrame = <function> defined @Aurora\Skin\FrameXML\BossBannerToast.lua:12
 AchievementButton_GetMiniAchievement = <function> defined @Aurora\SkinBlizzard_AchievementUI.lua:85
 AchievementButton_UpdatePlusMinusTexture = <function> defined @Aurora\SkinBlizzard_AchievementUI.lua:39
 AchievementFrameStats_SetStat = <function> defined @Aurora\SkinBlizzard_AchievementUI.lua:116
 NewPvpSeasonMixin = <table> {
 }
 UIDropDownMenuButton_OnClick = <function> defined @Aurora\Skin\SharedXML\UIDropDownMenu.lua:104
 PartyPoseRewardsMixin = <table> {
 }
 PlayerChoiceOptionFrameMixin = <table> {
 }
 LFGListUtil_SetSearchEntryTooltip = <function> defined @Aurora\Skin\FrameXML\LFGList.lua:130
 LFGCooldownCover_Update = <function> defined @Aurora\Skin\FrameXML\LFGFrame.lua:70
 PetStable_Update = <function> defined @Aurora\Skin\FrameXML\PetStable.lua:21
 CovenantSanctumMixin = <table> {
 }
 RaceFilter_OnHide = <function> defined @Aurora\SkinBlizzard_ArchaeologyUI.lua:16
 UIWidgetManagerMixin = <table> {
 }
 AuctionHouseFavoritesSearchButtonMixin = <table> {
 }
 GarrisonFollowerButton_AddAbility = <function> defined @Aurora\SkinBlizzard_GarrisonTemplates.lua:29
 CommunitiesTicketManagerDialogMixin = <table> {
 }
 CovenantRenownMixin = <table> {
 }
 EncounterJournal_ListInstances = <function> defined @Aurora\SkinBlizzard_EncounterJournal.lua:26
 OpenMail_Update = <function> defined @Aurora\Skin\FrameXML\MailFrame.lua:76
 InspectHonorFrame_Update = <function> defined @Aurora\SkinBlizzard_InspectUI.lua:45
 InboxFrame_Update = <function> defined @Aurora\Skin\FrameXML\MailFrame.lua:18
 AddonList_Update = <function> defined @Aurora\Skin\SharedXML\AddonList.lua:31
 PVPHonorXPBar_Update = <function> defined @Aurora\Skin\FrameXML\PVPHonorSystem.lua:14
 QuestFrame_SetTitleTextColor = <function> defined @Aurora\Skin\FrameXML\QuestFrame.lua:78
 ExhaustionTickMixin = <table> {
 }
 WardrobeSetsTransmogMixin = <table> {
 }
 WardrobeSetsCollectionMixin = <table> {
 }
 AchievementButton_GetMeta = <function> defined @Inter

### Expected Behavior

Not get bugsack errors

### Steps To Reproduce

1. Log In. That's it lol

**Comment by ZINK-ZINK** [comment]
Oh and I run A LOT of addons new and old, so there could easily be conflicts, but where the conflict is isn't obvious to me. Without Aurora loaded all my addons cooperate and I get no errors, so it's something specific to Aurora even if the actual problem is CovenantMissionHelper or another addon pissing off Aurora or something like that.

**Comment by ZINK-ZINK** [comment]
Thought Better Wardrobe and Transmog addon might've been the culprit so tried disabling that addon and reload. The Aurora error still showed up so that wasn't it.

**Comment by Gethe** [comment]
The stack trace mentions KalielsTracker, try disabling that. It hasn't been updated in a while.

**Comment by ZINK-ZINK** [comment]
@Gethe Ah yes Kaliels is likely the problem then. I had downloaded their Aurora Extension at https://www.curseforge.com/wow/addons/aurora-extension     so I ignored it as a possibility, but yea some of Kaliel's settings won't function now so it's probably their fault. Aesthetically it's a great quest tracker, but man does it have a habit of being buggy. TY I'll look for Kaliel's bug report site and forward this to them.
